src/test/java/org/opensearch/neuralsearch/processor/TextChunkingProcessorTests.java

```diff
@@ -1046,4 +1065,26 @@ public void testExecute_statsEnabled_withDelimiter_andSourceDataString_thenSucce
         assertEquals(1L, snapshots.get(EventStatName.TEXT_CHUNKING_PROCESSOR_EXECUTIONS).getValue().longValue());
         assertEquals(1L, snapshots.get(EventStatName.TEXT_CHUNKING_DELIMITER_EXECUTIONS).getValue().longValue());
     }
+
+    @SneakyThrows
+    public void testExecute_statsEnabled_withFixedCharLength_andSourceDataString_thenSucceed() {
```

Comment:
Rather than having this as a unit test, could you add the fixed char length stat tests to the existing IT [here](https://github.com/opensearch-project/neural-search/blob/575f77433397107d672a05ab0f4db3931cb21062/src/test/java/org/opensearch/neuralsearch/processor/TextChunkingProcessorIT.java#L174)? You should be able to reuse the already existing helpers, although it'd be helpful to add `INDEX_NAME3` for the new pipeline. That way we can test the the new info stat is counted correctly too